### DISCUSSION TITLE
Rebranding NEAR Wallet

### DISCUSSION THREAD

**Discussion by heycorwin** [near/near-wallet#2029] [Ideas] [answered]
As alluded to in recent team syncs, I've been putting some thought into the seemingly inevitable spinout of the wallet (timeline TBD) and what this means for the Wallet brand. There have been some discussions in the past about potentially rebranding the wallet to create some distance between the wallet product and NEAR Inc, to promote decentralization, etc... Now that it seems that there will be a spinout event with legal ramifications at some point in the near future, it feels appropriate to consider this again.

There are a couple relevant events occurring soon around the NEAR rebrand that @jakestutzman has been facilitating that could inform some decision making here, and I wanted to highlight them before offering a little proposal.

The NEAR brand refresh has been going on for some time now, and is slated to begin rolling out in the coming months. One of the ways this rebrand impacts our products is that all products such as the website, wallet, explorer, etc... would need to see some kind of refresh to bring them up to par with the visual design changes brought on by the new brand.

While this is exciting, it has the potential to act as a pivotal point for the Wallet brand. Updating our product UI, brand, and landing page to leverage the new brand elements is not an insignificant amount of work. We can do it, but if the spinout mentioned above truly is inevitable and could potentially occur within the next year, that means that we would then need to _repeat_ this process at that time in order to facilitate a brand transition for the Wallet to exist as its own entity, rather than one represented by NEAR Inc. (This would mean a name change, etc.)

To avoid double work within a short period of time, it may make sense to use this opportunity to go ahead and initiate this process. What I propose is that during the time between now and the rollout of the remainder of the NEAR brand refresh and website, we take the time to develop a "Minimum Viable Brand" (MVB) for the wallet that distinguishes it apart from NEAR Inc.

We would make some kind of announcement that "NEAR Wallet is rebranding to x" around the time of the NEAR brand refresh rollout, and launch the "MVB" and new name for the Wallet, with an updated landing page and visual updates across the UI. With this out of the way, the spinout event can occur with a focus on legal and logistics, without the need to repeat the effort to update the wallet brand more than once within the year.

This wouldn't result in any change for our users save for the brand update to start. It would simply be an announcement, a visual update, and a domain change. The rebranded wallet -- codename "Berry Wallet" -- would still exist as a property of NEAR Inc until the spinout was formalized, at which point legal ramifications or larger impact on the user base would be navigated and it would become a separate entity.

Would love to hear any thoughts around this that you all might have. @jakestutzman and I have discussed this already, and agree that this seems to make some sense, but I'm also curious to know if there are any glaring reason why we _shouldn't_ approach it this way. Thanks in advance for your participation!

cc @ilblackdragon @eriktrautman @stefanopepe

**Comment by eriktrautman** [comment]
Though at some distance from the product myself, the strategy makes sense to me.

**Comment by waverlymaven** [comment]
Would people still be able to reach it via wallet.near.org ? In an era of so many dubious crypto platforms and websites, I think it instills confidence in users to be able to visit that website. MetaMask did that for Ethereum (and as a browser extension it had its fair share of pirate versions/even got pushed out of the Google store at some point… since this is a web wallet I imagine it’ll be harder to spoof but you never know the kind of clones that can appear). Also all the Dapps that currently route people to NEAR wallet would have to get redirected proactively— so we don’t end up with a ton of broken links. 

Also how will consumers know “Berry Wallet” is specific to NEAR? There’s something really helpful given how early the space is right now about knowing that “NEAR wallet” is where you keep your NEAR… so finding other friendly ways to keep this accessible to the community under the new branding would be great :)

**Reply by heycorwin** [reply]
Great questions @waverlymaven! I'll try and address each one, and @stefanopepe or @ilblackdragon can weigh in with their thoughts around the transition specifically:

> Would people still be able to reach it via wallet.near.org ?

TBD. We would definitely need to facilitate a transition period and to take great care to shepherd users in the right direction. Ideally, we could manage this in such a way that wouldn't require much action from users themselves (devs perhaps). But yes, there would ultimately need to be some kind of transition strategy. We would need to be very intentional about displaying explicit messaging to users about the transition, and also have persistent messaging to act as signposts in places like wallet.near.org for the "set it and forget it" users who don't visit the wallet often.

> Also how will consumers know “Berry Wallet” is specific to NEAR? There’s something really helpful given how early the space is right now about knowing that “NEAR wallet” is where you keep your NEAR…

This is true! There are definitely some cons and risks associated with moving away from the NEAR brand, but ultimately I think there are more pros. The fact that NEAR Inc. currently owns and operates the only web wallet option makes it harder to boast decentralization, and having products like these be independently owned and operated is something that I think everyone benefits from both philosophically and legally/logistically. That's the ultimate driver in the spinout decision as far as I can tell, and this proposal is mostly just an effort to expedite the inevitable and to avoid double work. It's actually uncommon for crypto wallets to hold the brand of their parent chains, so I'm confident that people would be able to adapt easily enough.

> finding other friendly ways to keep this accessible to the community under the new branding would be great :)

100% This is in everyone's best interest and I think the new brand would take care to position itself as a staunch supporter and enabler of the NEAR Protocol and Community :)

**Comment by stefanopepe** [comment, marked as answer]
To weigh in, I agree to rebrand this project now, avoiding the double work later on.

I'm on track with defining the 2 years vision of the wallet (~30% complete), a task that once done will identify the personas and should be useful for this activity.

To answer @waverlymaven , and taking inspiration from Ethereum and other L1 networks:
- the current link would become a landing page for listing the various available wallets (even include near-cli!)
- any legacy login request may be invited to the new wallet address. 
In any case, there's a "wallet select" library in sight that will make this step unnecessary.

My concern is how we keep a tight alignment with NEAR Protocol during this growth phase since our almost-monopolistic role requires us (this new brand) to support _all_ use cases and personas at least until other wallets join the ecosystem.
Put in other words, Ethereum and other L1s offer wallets for very different segments: DeFi, NFTs, dApps, long-term custody - all with different tradeoffs in terms of onramp, convenience, security and UI. 
I feel confident to rebrand NEAR Wallet into something else, but what if the way to do better is to fork into more than one wallet (and brands)?
